javascript/javascript2/week3/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <script src="fastest_presser.js"></script>
```

Comment:
Here in your html file your using two JavaScript file, index.js in the body and fastest_presser.js in the header. This is not recommend, you have to choose if you want to put it in the header or in  the body. But the best practice the use JavaScript it is in the body. Your JavaScript file name is fatest_presser.js, but where index.js come from?